pymongo/database.py

```diff
@@ -342,7 +342,8 @@ def create_collection(
         All optional `create collection command`_ parameters should be passed
         as keyword arguments to this method. Valid options include, but are not
         limited to:
-
+          - ``checkExists`` (bool): if True (the default) , send a listCollections command to check
+            if the collection already exists before creation.
```

Comment:
checkExists is not a create command option so it should be an explicit check_exists argument.